build/includes/examples.mk

```
@@ -42,8 +42,9 @@ push-example-golang-images: example-image-push.supertuxkart
 push-example-golang-images: example-image-push.simple-game-server
 
 # Test to ensure the example image found in the % folder is on GAR. Fails if it is not.
-example-image-test.%:
-	$(DOCKER_RUN) bash -c "cd examples/$* && make gar-check"
+example-image-test.%: 
```

```suggestion
example-image-test.%:
```
Is there an extra space here?

---

Thanks for the review. I have deleted the extra space.